doc/content/design/coverage/index.md

```diff
@@ -98,7 +98,8 @@ terminated by receiving the `TERM` signal, a signal handler must be
 installed:
 
     let stop signal =
-      printf "caught signal %d\n" signal;
+      let name = Xapi_stdext_unix.Unixext.string_of_signal signal in
```

Comment:
This function got deleted.

Although I think most information in this markdown file is out of date.

Reply:
This is now using `Debug.pp.signal`